website/source/counters.md

```diff
@@ -1,5 +1,4 @@
-title: Counters
----
+## title: Counters
 
```

Comment:
This is probably fine, but I'm not sure I'm a fan of prettier updating markdown.

Reply:
we can omit markdown files pretty easily. i don’t really care either way, i mainly just care about consistency.